### ISSUE TITLE
Enable handle push deeplink

### ISSUE THREAD

**Issue by badoualy** [Appboy/appboy-segment-android#14]
Currently, since the integration is doing the init on the `Braze` singleton, and this can be done only once, we have no way of "cleanly" enabling the "handle push deeplink" feature.

In the mean time I did the following "hack":

```kotlin
application.getSharedPreferences("com.appboy.override.configuration.cache", 0).edit {
    putBoolean("com_appboy_handle_push_deep_links_automatically", true)
}
```

inside the integration ready callback to overwrite the value.

Would be nice to have the on/off toggle in the segment dashboard

**Comment by Bucimis** [comment]
@badoualy we recommend configuring Braze fields at compile time by creating an `appboy.xml` file in your `res/values` folder. Our native SDK sample app contains an example: https://github.com/Appboy/appboy-android-sdk/blob/master/droidboy/src/main/res/values/appboy.xml#L55 

let us know how it goes!
